### PR TITLE
Reverted change which removed loading text from KW splash

### PR DESCRIPTION
**Pull request by radujipa**

### CODE REVIEW

**Comment by radujipa** [comment]
[(2) As a user, I expect a loading indicator when launching KW](https://trello.com/c/mXK2Y1i4/1383-2-as-a-user-i-expect-a-loading-indicator-when-launching-kw)

File: debian/changelog [hunk 1]
```diff
@@ -1,8 +1,9 @@
 kano-profile (3.13.0-0) unstable; urgency=low
```
Comment (skarbat) [inline debian/changelog]: no version increase because it already has been for the upcoming release, right?

Comment (radujipa) [reply debian/changelog]: Yup, that's correct. This one is for 3.13 release.

Comment (skarbat) [reply debian/changelog]: Great! 👍

**Pull request merged**